com.ibm.streamsx.plumbing/com.ibm.streamsx.plumbing.filters/traceLevelFilter.spl

```diff
@@ -0,0 +1,29 @@
+/*
+# Licensed Materials - Property of IBM
+# Copyright IBM Corp. 2016
+*/
+namespace com.ibm.streamsx.plumbing.filters ;
+
+/**
+ * Dynamic filter that forwards data based on the trace level. 
```

Comment:
I guess I was expecting a more generic filter, that supported a parameter specifying the trace level that enabled the tuples to be passed? Is that possible?